src/cc_toolchain/BUILD

```diff
@@ -54,56 +54,57 @@ filegroup(
 )
 
 filegroup(
-    name = "clang_wrappers",
-    srcs = glob(["wrapper/clang-*"]),
+    name = "k8_gcc_wrappers",
+    srcs = glob(["wrapper/k8_gcc-*"]),
 )
 
 filegroup(
-    name = "clang_all",
+    name = "k8_gcc_all",
     srcs = [
-        ":clang_wrappers",
-        "@llvm_clang//:binaries",
-        "@llvm_clang//:clang_libs",
-        "@llvm_clang//:includes",
-        "@llvm_clang//:runtime_libs",
-        "@llvm_clang//:static_libs",
+        ":k8_gcc_wrappers",
+        "@k8_gcc//:binaries",
+        "@k8_gcc//:k8_gcc_libs",
+        "@k8_gcc//:includes",
+        "@k8_gcc//:runtime_libs",
+        "@k8_gcc//:static_libs",
     ],
 )
 
-cc_toolchain_config(
-    name = "clang-k8",
+cc_toolchain_config_k8(
+    name = "k8_gcc",
```

Comment:
so replace all instances of k8 with linux?

Reply:
A lot of the toolchain is general to  `k8` (really `x86_64` systems), but at this level we're linux specific because of the include directories. So not every instance, just here upwards

Reply:
ok done